### ISSUE TITLE
UnableToSampleException - when master fails in cluster.

### ISSUE THREAD

**Issue by FylmTM** [neo4j/neo4j#3535]
We have Neo4j cluster:
A - Master instance
B - Slave instance
C - Slave instance

If master (`A`) was killed (e.g. `kill -9 <pid>`), then new master election was performed. After awhile `B` becomes master instance.
Then, we try to bootup A instance. So, this instance was master before and now joins cluster as slave.
`A` instance decides that he has wrong database version and downloads up-to-date version from master (`B`).

And there is where error occurs. 

```
org.neo4j.server.rrd.UnableToSampleException
    at org.neo4j.server.rrd.sampler.NodeIdsInUseSampleable.getValue(NodeIdsInUseSampleable.java:47)
    at org.neo4j.server.rrd.RrdSamplerImpl.updateSample(RrdSamplerImpl.java:59)
    at org.neo4j.server.rrd.RrdJob.run(RrdJob.java:43)
    at org.neo4j.server.rrd.ScheduledJob$1.run(ScheduledJob.java:41)
    at java.util.TimerThread.mainLoop(Timer.java:555)
    at java.util.TimerThread.run(Timer.java:505)
```

We see this when log download is in process/finished. And this error continues to show endlessly:

```
2014-11-13 12:11:50.147+0000 INFO  [Cluster] Checking store consistency with master
2014-11-13 12:11:50.853+0000 INFO  [Cluster] The store is inconsistent. Will treat it as branched and fetch a new one from the master
2014-11-13 12:11:52.030+0000 INFO  [Cluster] Copying store from master
... copying store
... copying schema
... copying logs
2014-11-13 12:13:57.768+0000 INFO  [API] Server started on: http://0.0.0.0:7474/
// it starts server, while logs download is in progress
... org.neo4j.server.rrd.UnableToSampleException error loop
```

If we restart this instance, then everything is OK. 

So, in summary: we get `UnableToSampleException` error when try to bootup instance, that was master before, for first time.

Any suggestions?

--------

Neo4j: 2.1.5 Enterprise
OS: CentOS 6.5
